native/src/components/FeedbackContainer.tsx

```diff
@@ -33,7 +41,10 @@ const FeedbackContainer = ({ query, language, routeType, cityCode, slug }: Feedb
 
   useEffect(() => {
     setSearchTerm(query)
-  }, [query])
+    if (noResults === true) {
```

Comment:
```suggestion
    if (noResults) {
```
it's already a boolean